scheduler/src/cook/mesos/constraints.clj

```diff
@@ -186,11 +187,17 @@
 ;; Group host placement constraints
 
 (defn get-cotasks
-  [db group job]
+  [db group job cotask-cache]
```

Comment:
Nitpick: I prefer `group-uuid->running-tasks-cache` as it tells the reader and caller about the mapping of the cache. Also, the cache contains the _running_ cotasks.